lib/rspec/concepts/example_group.rb

```diff
@@ -0,0 +1,60 @@
+module RSpec
+  module Concepts
+    module ExampleGroup
```

Comment:
This is identical to https://github.com/apotonick/rspec-cells/blob/master/lib/rspec/cells/example_group.rb. We should rather reuse the existing (same with the other files that you copied). Can you do that? :beer: